src/python/grpcio/support.py

```diff
@@ -28,18 +28,20 @@
 """
 C_PYTHON_DEV_ERROR_MESSAGE = """
 Could not find <Python.h>. This could mean the following:
-  * You're on Ubuntu and haven't run `apt-get install python-dev`.
-  * You're on RHEL/Fedora and haven't run `yum install python-devel` or
-    `dnf install python-devel` (make sure you also have redhat-rpm-config
+  * You're on Ubuntu and haven't run `apt-get install <PY_REPR>-dev`.
+  * You're on RHEL/Fedora and haven't run `yum install <PY_REPR>-devel` or
+    `dnf install <PY_REPR>-devel` (make sure you also have redhat-rpm-config
     installed)
   * You're on Mac OS X and the usual Python framework was somehow corrupted
     (check your environment variables or try re-installing?)
   * You're on Windows and your Python installation was somehow corrupted
     (check your environment variables or try re-installing?)
 """
+PYTHON_REPRESENTATION = 'python' if sys.version_info[0] == 2 else 'python3'
```

Comment:
Nit: You could make this Python 4-proof and do three cases: Python 2, and Python 3, and anything else (error).

Reply:
Updated.

If 4.0 will simply be the version after 3.9, I wonder if there will be a `python4-dev` package. It's possible that they reuse `python3-dev` as the name for the Python header package.

Reply:
Possible. But there's no way to know until they make the decision. Hopefully, they'll never have to 🤞, but forever is a long time.